play.rb

```diff
@@ -0,0 +1,100 @@
+$LOAD_PATH << '.'
+require 'deck'
+require 'dealer'
+require 'player'
+
+def check_bust(player, dealer)
+  if player.total > 21
+    puts 'PLAYER BUST!! Dealer wins! '\
+    "Player total: #{player.total}, Dealer total: #{dealer.total}"
+    end_game
+  end
+  if dealer.total > 21
+    puts 'DEALER BUST!! Player wins! '\
+    "Player total: #{player.total}, Dealer total: #{dealer.total}"
+    end_game
+  end
+end
```

Comment:
What do you think about moving the "bust" logic into the Player class? Dealer would inherit this behavior.  

You could add a method like:

```ruby
class Player

   def busted?
       total > 21
   end

end

# then you could check...

if player.busted?
```